### PR TITLE
feat: adds most of the pricing

### PR DESCRIPTION
**Pull request by R-K-H**
need to fix units to match lot size of quote or something

Potential for merge, with some rough edges. But could at least be a great start for someone who knows how to fix it :)

- [ ] padding / margin for the order book display is out of whack
- [ ] settle funds tx gets built and sent to the wallet, but fails simulation
- [x] explorer selector doesn't do anything
- [ ] orders in various statuses are displayed, but unsure if they're cancelled or filled
- [ ] likely some shit I'm not good at janky left over

For the orders, we can get pricing from the open orders, my assumption is you'll have to do something else with the account state given it's a trade or cancelled or filled etc.

For the settle funds I think the error is with `userBaseAccount` and `userQuoteAccount` I'm not sure how to determine that.

For the explorer selection, it's just something where we need to fetch the configured value and append it with proper `tx/` or `account/` to make it behave.

### CODE REVIEW

**Comment by R-K-H** [comment]
Good for a review and then merge, it's almost functioning, someone with better knowhow will be able to solve it.

File: components/Proposals/ProposalDetailCard.tsx [hunk 1]
```diff
@@ -461,7 +488,15 @@ export function ProposalDetailCard({ proposalNumber }: { proposalNumber: number
                     order.account.position.asksBaseLots,
                   );
                   return (
-                    <Table.Tr key={order.publicKey.toString()}>
+                    (
+                      (order.account.openOrders[0].isFree === 0)
+                    ) ? (
+                     <Table.Tr key={order.publicKey.toString()}>
+                      <Table.Td>
+                        <a href={`https://solana.fm/accounts/${order.publicKey.toString()}`} target="_blank" rel="noreferrer">
```
Comment (Dodecahedr0x) [inline components/Proposals/ProposalDetailCard.tsx:496]: You forgot to use the explorer hook?

File: hooks/useExplorerConfiguration.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  Xray = 'xray',
+  Solana = 'solana'
+}
+
+export function useExplorerConfiguration() {
+  const [explorer, setExplorer] = useLocalStorage<Explorers>({
+    key: 'meta-dao-explorer-configuration',
+    defaultValue: Explorers.SolanaFM,
+  });
+  const endpoint = useMemo(() => {
```
Comment (Dodecahedr0x) [inline hooks/useExplorerConfiguration.ts:16]: I think most of them have some additional formatting needed (e.g. `explorer.solana.com/address` or `solscan.io/account`.
The explorer hook could also take as input the account or transaction hash and output the correct explorer link

Comment (R-K-H) [reply hooks/useExplorerConfiguration.ts:16]: Will work to resolve this.

File: hooks/useOpenbookTwap.ts [hunk 3]
```diff
@@ -115,6 +117,43 @@ export function useOpenbookTwap() {
...
   );
 
+  const settleFundsTransactions = useCallback(
+    async (orderId: BN, market: MarketAccountWithKey) => {
+      if (!wallet.publicKey || !wallet.signAllTransactions || !openbook || !openbookTwap) {
+        return;
+      }
+      console.log(market.publicKey.toString());
+      const openOrdersAccount = findOpenOrders(orderId, wallet.publicKey);
+      const userBaseAccount = getAssociatedTokenAddressSync(
```
Comment (R-K-H) [inline hooks/useOpenbookTwap.ts:127]: Here's the issue point for getting it.

**Pull request merged**